go.mod

```
@@ -2,27 +2,39 @@ module github.com/grafana/xk6-http
 
 go 1.22.0
 
-require go.k6.io/k6 v0.57.0
+require (
+	github.com/google/uuid v1.6.0
+	github.com/grafana/sobek v0.0.0-20241024150027-d91f02b05e9b
+	github.com/stretchr/testify v1.10.0
+	go.k6.io/k6 v0.54.1-0.20241025083358-192a49e1c20d
```

```suggestion
	go.k6.io/k6 v0.54.1-0.20241025083358-192a49e1c20d
```
This is not expected

---

@codebien Thanks for noticing.

There is no `httpMultiBin` at `k6 v0.57.1-0` and needed that for writing tests.
Do you have any recommendation on that?
The corresponding error i encounter after running `go mod tidy`
```bash
go: finding module for package go.k6.io/k6/lib/testutils/httpmultibin
go: github.com/grafana/xk6-http/pkg/http tested by
        github.com/grafana/xk6-http/pkg/http.test imports
        go.k6.io/k6/lib/testutils/httpmultibin: module go.k6.io/k6@latest found (v0.57.0), but does not contain package go.k6.io/k6/lib/testutils/httpmultibin
```

---

Hi @codebien, I looked at k6 files and found out the httpmultibin package moved under internal which can't be imported from this project,
Was there any reason behind that?
Should i write the test cases another way?